docs/sources/datasources/tempo/query-editor/_index.md

```diff
@@ -26,86 +26,191 @@ refs:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/datasources/tempo/service-graph/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/service-graph/
+  recorded-queries:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/recorded-queries/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/recorded-queries/
+  query-history-management:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/explore/query-management/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/explore/query-management/
+  query-inspector:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/explore/explore-inspector/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/explore/explore-inspector/
 ---
 
 # Query tracing data
 
 The Tempo data source's query editor helps you query and display traces from Tempo in [Explore](ref:explore).
+The queries use [TraceQL](/docs/tempo/latest/traceql), the query language designed specifically for tracing.
 
-This topic explains configuration and queries specific to the Tempo data source.
-For general documentation on querying data sources in Grafana, see [Query and transform data](ref:query-transform-data).
+For general documentation on querying data sources in Grafana, refer to [Query and transform data](ref:query-transform-data).
 
-To add TraceQL panels to your dashboard, refer to the [Traces panel documentation](/docs/grafana/latest/panels-visualizations/visualizations/traces/).
+## Before you begin
 
-To learn more about Grafana dashboards, refer to the [Use dashboards documentation](/docs/grafana/latest/dashboards/use-dashboards/).
+You can compose TraceQL queries in Grafana and Grafana Cloud using **Explore** and a Tempo data source.
 
-## Write TraceQL queries in Grafana
+### TraceQL knowledge helpful, but not required
 
-You can compose TraceQL queries in Grafana and Grafana Cloud using **Explore** and a Tempo data source. You can use either the **Query type** > **Search** (the TraceQL query builder) or the **TraceQL** tab (the TraceQL query editor).
-Both of these methods let you build queries and drill-down into result sets.
+You don't have to know TraceQL to create a query.
+You can use the **Search** query builder's user interface to select options to search your data.
+These selections generate a TraceQL query.
+Any query generated in Build mode can be transferred to the Code mode, where you can edit the query directly.
 
 To learn more about how to query by TraceQL, refer to the [TraceQL documentation](/docs/tempo/latest/traceql).
 
-### TraceQL query builder
+## Choose a query editing mode
+
+The query editor has three modes, or **Query types**, that you can use to explore your tracing data.
```

Comment:
This was also a left over from a rewrite I reverted. I'm glad you're reading this so it can be clarified. 
```suggestion
The query editor has three **Query types** that you can use to explore your tracing data.
```